spec/models/intake_spec.rb

```diff
@@ -743,23 +743,10 @@
     let(:source) { nil }
     let(:intake) { create :intake, state_of_residence: state, source: source }
 
-    before(:all) do
```

Comment:
It's simpler to not bother with `before(:all)`, so I went for `before do` instead. This doesn't seem to cause significant slowdowns.